src/player/player.rs

```diff
@@ -223,6 +223,8 @@ playlist-modify-public,\
 playlist-modify-private,\
 streaming";
 
+const KNOWN_AP_PORTS: &[u16] = &[80, 443, 4070];
```

Comment:
You can have your Option there, and have it be owned as well since there are no heap allocations needed:

```rust
const KNOWN_AP_PORTS: [Option<u16>; 4] = [None, Some(80), Some(443), Some(4070)];
```

Reply:
Great. Fixed in the latest commit.